scripts/update-docs.sh

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# This is temporary script to update the docs while our CI is disabled due to 
+# security.
```

Comment:
Would avoid calling too much attention in a public repo.

Reply:
ah right i can jst remove